### ISSUE TITLE
unused & deprecated dependency? Please remove if so.

### ISSUE THREAD

**Issue by Loewenhardt** [kikinteractive/kik-node#45]
### Expected behavior
No warnings when installing the package.
### Actual behavior
`warning @kikinteractive/kik > node-uuid@1.4.7: Use uuid module instead`
### Enviromen`t Details
`> yarn list @kikinteractive/kik`
yarn list v0.22.0
└─ @kikinteractive/kik@2.2.0
### Steps to reproduce the behaviour
1. yarn add @kikinteractive/kik

I wanted to upgrade it and create a pull request, but it's probably an old dependency which isn't used anywhere. Am I right or it's something I didn't see? If you don't need it, please remove it. Thank you.

**Comment by remvst** [comment]
Good catch, here is a fix https://github.com/kikinteractive/kik-node/pull/47
Will probably wait for other additions before pushing that to npm.
